examples/load_fw_canopen.py

```diff
@@ -0,0 +1,73 @@
+from typing import Any, Dict
+
+from ingenialink import CAN_BAUDRATE, CAN_DEVICE
+from ingenialink.exceptions import ILFirmwareLoadError
+
+from ingeniamotion.motion_controller import MotionController
+
+
+def status_log(current_status: str) -> None:
+    print(f"Load firmware status: {current_status}")
+
+
+def progress_log(current_progress: int) -> None:
+    print(f"Load firmware progress: {current_progress}%")
+
+
+def load_firmware_canopen(can_drive: Dict[str, Any]) -> None:
+    mc = MotionController()
+    # Find available nodes
+    node_id_list = mc.communication.scan_servos_canopen(
+        can_drive["device"],
+        baudrate=can_drive["baudrate"],
+        channel=can_drive["channel"],
+    )
+    if can_drive["node_id"] not in node_id_list:
+        print(f"Node {can_drive['node_id']} is not detected.")
+        return
+    else:
+        print(f"Found nodes: {node_id_list}")
+
+    print("Starts to establish a communication.")
+    mc.communication.connect_servo_canopen(
+        can_drive["device"],
+        can_drive["dictionary_path"],
+        can_drive["node_id"],
+        baudrate=can_drive["baudrate"],
+        channel=can_drive["channel"],
+    )
+    if "default" in mc.servos:
```

Comment:
should probably be try / catch instead

Reply:
I'm going to remove this conditional statement because the no connection state can't be reached.